src/libYARP_OS/include/yarp/os/Value.h

```diff
@@ -276,6 +276,16 @@ class YARP_OS_API yarp::os::Value : public Portable, public Searchable {
         return asString() != alt;
     }
 
+    bool operator==(const int alt) const
```

Comment:
Additional considerations for multipurpose data-type like value: operator == always generates problems and should be generally avoided. What happens for example if the value internally stored was a double? With this commit you will allow users to compare it with an int. The comparison is ok (more or less), but I fear it may generate other misusage problems in the future...